pkg/blob/controllerserver.go

```diff
@@ -118,6 +119,30 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 			if strings.EqualFold(v, trueValue) {
 				isHnsEnabled = pointer.Bool(true)
 			}
+		case softDeleteBlobsField:
+			if v != "" {
+				days, err := strconv.Atoi(v)
+				if err != nil {
+					return nil, status.Errorf(codes.InvalidArgument, fmt.Sprintf("invalid %s:%s in storage class", softDeleteBlobsField, v))
+				}
+				if days <= 0 || days > 365 {
+					return nil, status.Errorf(codes.InvalidArgument, fmt.Sprintf("invalid %s:%s in storage class, should be in range [1, 365]", softDeleteBlobsField, v))
+				}
+				softDeleteBlobs = int32(days)
+			}
+		case softDeleteContainersField:
+			if v != "" {
+				days, err := strconv.Atoi(v)
```

Comment:
this is duplicated with L124 - L131, could you create a new function to parse the days num?